index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title >Fork Me! FCC: Test Suite Template</title>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css" >
+</head>
 
-    <link rel="stylesheet" href="style.css">
-
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<body>
+   <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
    
     <div class="container">
```

Comment:
- Kindly do not add blank lines and spaces without a reason.

Reply:
- [ ] Great job doing the review but I think here it would be great if you add the reason because maybe it is obvious to you but not for the others ✅👌🏻👌🏻💯💯